src/reputation/tasks.py

```diff
@@ -31,6 +34,8 @@
 
 DEFAULT_REWARD = 1000000
 
+TRANSACTION_AGE_LIMIT = 60 * 60 * 6  # 6 hours
```

Comment:
nit^2: Can we change to something like `RECENT_TRANSACTION_LIMIT` or `RECENT_TRANSACTION_TTL` ? I wasn't sure what this meant without looking at it in context

Reply:
I will call it `PENDING_TRANSACTION_TTL`. What do you think?

Reply:
Sounds good